### PR TITLE
how to scale doc

### PR DESCRIPTION
**Pull request by katamaks**

### CODE REVIEW

File: developerportal/howto/scale-environment.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+title: "Scaling Your Environment in Mendix Cloud v4"
+space: "Developer Portal"
+category: "Deploy"
+description: "Describes how to scale your environments in Mendix Cloud v4."
+tags: ["Scale","Environments","Mendix Cloud","Developer Portal"]
+---
+
+## 1 Introduction
+
+To scale an evironment of your app, you will need to have a licensed app. All apps that are becoming licensed after April 2017, will be by default hosted in Mendix Cloud Version 4.
```
Comment (jtwaleson) [inline developerportal/howto/scale-environment.md]: a licensed app in the Enterprise plan

File: developerportal/howto/scale-environment.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+**This how-to will teach you how to do the following:**
+
+* Scale your environment in Mendix Cloud v4
+
+# 2 Prerequisites
+
+Before starting this how-to, make sure you have completed the following prerequisites:
+
+* You have a licensed app hosted in Mendix Cloud v4
+* The runtime of the app is Mendix 7.0 or higher
```
Comment (jtwaleson) [inline developerportal/howto/scale-environment.md]: runtime -> Mendix version

File: developerportal/howto/scale-environment.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## 3 Mendix Cloud Version
+
+To see in which cloud version your app is hosted on:
```
Comment (jtwaleson) [inline developerportal/howto/scale-environment.md]: is hosted (remove on)

File: developerportal/howto/scale-environment.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+3. Click **Details** of a selected environment.
+
+    ![](attachments/deploy/environment-details.png)
+
+4. Next to **Mendix Cloud Version** you will see the version of the cloud that your licensed app is hosted on.
+
+The possible cloud versions are:
+
+*   Mendix Cloud v3 - no scaling possible
```
Comment (jtwaleson) [inline developerportal/howto/scale-environment.md]: no scaling possible -> only vertical scaling

File: developerportal/howto/scale-environment.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+*   Mendix Cloud v4 - scaling is possible
```
Comment (jtwaleson) [inline developerportal/howto/scale-environment.md]: scaling -> vertical and horizontal scaling

File: developerportal/howto/scale-environment.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+If you have more than 1GB (for example 16GB) RAM of the **Total Allocated Memory**, you can spread the memory among 4 **instances**. 
+
+For example, you can scale 16GB of the **Total Allocated Memory** four times with 4GB over 4 **instances** (4 x 4GB = 16GB).
+
+### 4.2 Scaling Example 2
+
+You don't need to use the whole memory at once. You can just use part of it and then use the rest of the memory later.
+
+For example, you can use one **instance** and store 4GB of RAM in there. The other 12GB of RAM will be unused.
```
Comment (jtwaleson) [inline developerportal/howto/scale-environment.md]: store -> utilize

File: developerportal/howto/scale-environment.md (hunk 1 above)
Comment (jtwaleson) [inline developerportal/howto/scale-environment.md]: All licensed apps created after April 2017,

File: developerportal/howto/scale-environment.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Scaling Your Environment in Mendix Cloud v4"
+space: "Developer Portal"
+category: "Deploy"
+description: "Describes how to scale your environments in Mendix Cloud v4."
+tags: ["Scale","Environments","Mendix Cloud","Developer Portal"]
+---
+
+## 1 Introduction
+
```
Comment (jtwaleson) [inline developerportal/howto/scale-environment.md]: Apps in the Mendix Cloud can be scaled n various ways. Memory can be added to the application container. This is called vertical scaling. Mendix 7+ apps running on Mendix Cloud v4 can be scaled horizontally as well, in that case multiple runtime containers will run simultaneously. Incoming traffic for your app is randomly distributed over the running instances. Only the first instance will excecute **Scheduled events**.

File: developerportal/howto/scale-environment.md (hunk 1 above)
Comment (katamaks) [inline developerportal/howto/scale-environment.md]: done

File: developerportal/howto/scale-environment.md (hunk 7 above)
Comment (katamaks) [inline developerportal/howto/scale-environment.md]: done

File: developerportal/howto/scale-environment.md (hunk 2 above)
Comment (katamaks) [inline developerportal/howto/scale-environment.md]: done

File: developerportal/howto/scale-environment.md (hunk 3 above)
Comment (katamaks) [inline developerportal/howto/scale-environment.md]: done

File: developerportal/howto/scale-environment.md (hunk 4 above)
Comment (katamaks) [inline developerportal/howto/scale-environment.md]: done

File: developerportal/howto/scale-environment.md (hunk 5 above)
Comment (katamaks) [inline developerportal/howto/scale-environment.md]: done

File: developerportal/howto/scale-environment.md (hunk 6 above)
Comment (katamaks) [inline developerportal/howto/scale-environment.md]: done

**Comment by katamaks** [comment]
@jtwaleson please approve review

File: content/developerportal/howto/scale-environment.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Scaling Your Environment in Mendix Cloud v4"
+category: "Deploy"
+description: "Describes how to scale your environments in Mendix Cloud v4."
+tags: ["Scale","Environments","Mendix Cloud","Developer Portal"]
+---
+
+## 1 Introduction
+
+Licensed apps in the Mendix Cloud can be scaled n various ways. Memory can be added to the application container. This is called vertical scaling. Mendix 7+ apps created after April 2017 and running on Mendix Cloud v4 can be scaled horizontally as well, in that case multiple runtime containers will run simultaneously. Incoming traffic for your app is randomly distributed over the running instances. Only the first instance will excecute [Scheduled events](/refguide/scheduled-events).
```
Comment (jtwaleson) [inline content/developerportal/howto/scale-environment.md]: 'n' -> 'in'

Comment (jtwaleson) [inline content/developerportal/howto/scale-environment.md]: remove 'created after April 2017'. You just need Mendix 7 and Cloud v4.

**Comment by katamaks** [comment]
@jtwaleson please approve

**Pull request merged**